ibis/backends/dask/tests/execution/test_arrays.py

```diff
@@ -3,6 +3,7 @@
 import dask.dataframe as dd
 import numpy as np
 import pandas as pd
+import pandas._testing as ptm
```

Comment:
I didn't think this was an `_testing` import but I see the pandas backend is the same.